WordPress/Classes/ViewRelated/Reader/ReaderPostCardCell.swift

```diff
@@ -357,53 +291,54 @@ import WordPressShared
         }
 
         let request = NSMutableURLRequest(URL: requestURL)
-
-        let acctServ = AccountService(managedObjectContext: ContextManager.sharedInstance().mainContext)
-        if let account = acctServ.defaultWordPressComAccount() {
-            let token = account.authToken
-            let headerValue = String(format: "Bearer %@", token)
-            request.addValue(headerValue, forHTTPHeaderField: "Authorization")
+        guard let headerAuth = imageHeaderAuthorization else {
+            return request
         }
-
+        request.addValue(headerAuth, forHTTPHeaderField: "Authorization")
         return request
     }
 
     private func configureTitle() {
         if let title = contentProvider?.titleForDisplay() {
-            let attributes = WPStyleGuide.readerCardTitleAttributes() as! [String: AnyObject]
+            let attributes = readerCardTitleAttributes as! [String: AnyObject]
             titleLabel.attributedText = NSAttributedString(string: title, attributes: attributes)
-            titleLabelBottomConstraint.constant = titleLabelBottomConstraintConstant
-
+            if titleLabel.hidden {
```

Comment:
This was originally for performance, but based on a false assumption. I'm removing these checks.